avocado/core/result.py

```diff
@@ -188,7 +188,7 @@ def end_test(self, state):
         :type state: dict
         """
         self.tests_run += 1
-        self.total_time += state['time_elapsed']
+        self.total_time += state.get('time_elapsed', 0)
```

Comment:
I'd go with `-1` as it clearly shows that represents an unknown, default value, along the lines of `<unknown>`.  Zero total time can be ambiguous as either a real value for a quick test or a default value.